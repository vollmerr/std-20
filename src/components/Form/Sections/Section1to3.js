import React from 'react';
import { FieldCheckboxes, FieldDate, FieldInput } from 'state-template';

import schema from '../schema';

const Section1to3 = () => (
  <div className={'row'}>
    <div className={'col-md-6'}>
      <div className={'row'}>
        <FieldInput {...schema.agencyRequestNumber} className={'col-md-6'} />
        <FieldDate {...schema.requestDate} className={'col-md-6'} />
      </div>

      <FieldInput {...schema.contractorName} />
    </div>

    <div className={'col-md-6'}>
      <FieldCheckboxes {...schema.requestType} />
    </div>
  </div>
);

export default Section1to3;
