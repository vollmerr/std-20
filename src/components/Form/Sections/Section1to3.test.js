import React from 'react';
import { shallow } from 'enzyme';

import Section1to3 from './Section1to3';

const props = {};

let wrapper;
describe('Section1to3', () => {
  beforeEach(() => {
    wrapper = shallow(<Section1to3 {...props} />);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
