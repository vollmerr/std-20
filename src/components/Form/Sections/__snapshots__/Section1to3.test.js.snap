// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section1to3 should render correctly 1`] = `
<div
  className="row"
>
  <div
    className="col-md-6"
  >
    <div
      className="row"
    >
      <Component
        className="col-md-6"
        label="1. Agency Request Number"
        name="agencyRequestNumber"
      />
      <Component
        className="col-md-6"
        label="2. Date"
        name="requestDate"
      />
    </div>
    <Component
      label="Contractor Name"
      name="contractorName"
    />
  </div>
  <div
    className="col-md-6"
  >
    <Component
      helpText="Check all boxes that apply to this request"
      label="3. Request Type"
      name="requestType"
      options={
        Array [
          Object {
            "label": "Service",
            "value": "service",
          },
          Object {
            "label": <React.Fragment>
              Equipment (Requires a 
              <A
                href="https://cdt.ca.gov/services/calnet-ordering/"
                text="Form 65"
                to=""
              />
              )
            </React.Fragment>,
            "value": "equipment",
          },
          Object {
            "label": "Other",
            "value": "other",
          },
        ]
      }
    />
  </div>
</div>
`;
